app/controllers/country_controller.rb

```diff
@@ -101,4 +101,11 @@ def load_essential_vars
     @country_presenter = CountryPresenter.new(@country)
     @tab_presenter = TabPresenter.new(@country)
   end
+
+  private 
+
+  def flag_path country_name
```

Comment:
I'm in favour of putting brackets around the argument just to make it more explicit what is being passed in.

Reply:
Will make this change now